### PR TITLE
[CHE-205]: Fix bug where we are seeing cluster name encoded

### PR DESCRIPTION
**Pull request by bswetalina**
This is related to issue : https://github.com/CloudHealth/helm/issues/28

### CODE REVIEW

**Review by gm-cht — APPROVED** [review]

**Review by gm-cht — CHANGES_REQUESTED** [review]
New Version + deprecated: true flag is needed in Chart.yaml.
https://helm.sh/docs/topics/charts/#the-chartyaml-file

**Review by kscherme — APPROVED** [review]

**Comment by bswetalina** [comment]
> New Version + deprecated: true flag is needed in Chart.yaml. https://helm.sh/docs/topics/charts/#the-chartyaml-file

I am not sure why but it is overwriting the deprecated flag so for now I have created a new version

File: index.yaml [hunk 1]
```diff
@@ -19,7 +35,23 @@ entries:
     version: 1.0.2
   - apiVersion: v2
     appVersion: 2.0.175
-    created: "2022-03-09T10:03:25.702938-05:00"
+    created: "2022-04-05T19:57:29.895651Z"
```
Comment (gm-cht) [inline index.yaml:38]: Can you revert this change to 1.0.2?

**Review by gm-cht — CHANGES_REQUESTED** [review]

**Review by gm-cht — APPROVED** [review]
